maint/github_weekly_meeting.py

```diff
@@ -0,0 +1,123 @@
+#! /usr/bin/env python
+
+import sys, os
+from datetime import datetime, timedelta
+from pprint import pprint
+from getpass import getpass
+
+from github3 import login
+import github3
+
+
+def fetch(target_repo, last_num, gh):
+    orgname, reponame = target_repo.split('/', 1)
```

Comment:
I think these should just come in as parameters since `target_repo` is never used

Reply:
fixed in f681a847495e10b3e6f8e2eb71a2f233764ba298